istioctl/pkg/version/version.go

```diff
@@ -75,14 +76,20 @@ func getRemoteInfo(ctx cli.Context, opts clioptions.ControlPlaneOptions) (*istio
 func getRemoteInfoWrapper(ctx cli.Context, pc **cobra.Command, opts *clioptions.ControlPlaneOptions) func() (*istioVersion.MeshInfo, error) {
 	return func() (*istioVersion.MeshInfo, error) {
 		remInfo, err := getRemoteInfo(ctx, *opts)
+		var errMses []string
+
+		if remInfo == nil {
+			errMses = append(errMses, "Istio is not present in the cluster")
+		}
+
 		if err != nil {
-			fmt.Fprintf((*pc).OutOrStderr(), "%v\n", err)
-			// Return nil so that the client version is printed
-			return nil, nil
+			errMses = append(errMses, fmt.Sprintf("-- %v", err))
```

Comment:
nit: I would make it `error1: error2: error3` not `--`. `:` is standard Go practice

Reply:
should we just use `errors.Join` here?

Reply:
ack